demo_exam/tasks.h

```diff
@@ -0,0 +1,64 @@
+

```

Comment:
I'm curious - how much time did you need to solve the tasks? :)

Reply:
It took me aprox. 30 minutes. Will make sure to use the extra time for torough testing on the actual exam! 😄